tkey/src/main/java/com/web3auth/tkey/ThresholdKey/Modules/TSSModule.java

```diff
@@ -88,6 +95,13 @@ public static void getTSSPubKey(ThresholdKey thresholdKey, String tssTag, Thresh
         });
     }
 
+    /**
+     * returns list of all the tss tags for given threshold key.
+     * @param thresholdKey The threshold key to act on.
+     * @throws RuntimeError Indicates invalid parameters were used.
+     * @throws JSONException Indicates invalid json parameters were used.
```

Comment:
These can be made more specific.

The RuntimeError is an invalid ThresholdKey.
The JSONException however is not from a valid parameter but the library returning invalid json (shouldn't happen ever though)

Reply:
done